scripts/submit_eval_jobs.py

```diff
@@ -633,14 +631,33 @@ def adjust_gpus(task_spec, experiment_group, model_name, gpu_multiplier):
         oe_eval_cmd += f" --run-id {args.run_id}"
     if args.step:
         oe_eval_cmd += f" --step {args.step}"
+    
     # add string with number of gpus
-    num_gpus = task_spec['resources']['gpuCount']
-    # if num_gpus > 1, double it again for oe-eval configs
-    # open_instruct GPT adjustment wasn't quite enough
-    # adjusted here so the GPU configs in open-instruct eval are not impacted by the change
-    # tested reasonably extensively with 70B
-    if num_gpus > 1:
-        num_gpus *= 2
+    if args.oe_eval_task_suite == 'SAFETY_EVAL' or args.oe_eval_task_suite == 'SAFETY_EVAL_REASONING':
```

Comment:
Discussed offline.
Let's remove the custom logic here and make it inline with everything else, but then for SAFETY_EVAL_REASONING specifically just double the num gpus.